File: frontend/src/lib/brand.ts
/**
 * The platform's name, in one place.
 *
 * WHY THIS FILE EXISTS
 *
 * The name was written out by hand wherever it appeared, and it drifted. On
 * 2026-08-26 the landing page alone carried three different Arabic names:
 *
 *   منصة رحلة المورد البشري الإماراتي     (the footer)
 *   منصة تنمية الموارد البشرية الإماراتية  (the invitation email)
 *   منصة "إماراتي" للتنمية البشرية        (the correct one)
 *
 * A platform that cannot spell its own name consistently is not a cosmetic
 * problem when the name is the first thing a candidate sees in an email from a
 * government body they have never heard from before.
 *
 * THE QUOTES ARE PART OF THE NAME. "Emirati" / "إماراتي" is the product name
 * being quoted inside the descriptive title — it is not emphasis and not a
 * typo, and it must survive translation, truncation and copy-editing.
 */

/** English: "Emirati" Human Development Platform */
export const PLATFORM_NAME_EN = '"Emirati" Human Development Platform';

/** Arabic: منصة "إماراتي" للتنمية البشرية */
export const PLATFORM_NAME_AR = 'منصة "إماراتي" للتنمية البشرية';

/** Pick the name for the language currently on screen. */
export const platformName = (language: string): string =>
    language === 'ar' ? PLATFORM_NAME_AR : PLATFORM_NAME_EN;

/**
 * The Council is a different body from the platform it runs, so it does NOT
 * carry the quotes — those belong to the product name alone.
 *
 * Its English name was wrong here until 2026-08-26: this file said "Emirati
 * Human Development Council", omitting "Resources". It was caught because the
 * invitation email and the Exchange signature appeared one above the other in
 * the same message and disagreed. Confirmed by the owner; note that DGHR's own
 * signature says "Resource" singular, and the Arabic — مجلس تنمية الموارد
 * البشرية الإماراتية — is plural, which is what settles it.
 */
export const COUNCIL_NAME_EN = 'Emirati Human Resources Development Council';
export const COUNCIL_NAME_AR = 'مجلس تنمية الموارد البشرية الإماراتية';

File: backend/brand.py
"""The platform's name and the Council's, in one place.

Mirrors `frontend/src/lib/brand.ts`. Two modules now compose email that names
the platform — NAFIS seeker invitations and company magic links — and a name
copied into a second file is a name that will eventually differ from the first.

THE QUOTES ARE PART OF THE PLATFORM'S NAME. "Emirati" / "إماراتي" is the
product name quoted inside the descriptive title; it is not emphasis, and it
must survive translation, truncation and copy-editing.

Before this was pinned, the landing page and the invitation email carried FIVE
different Arabic names between them. That is not cosmetic when the name is the
first thing a candidate or an employer sees in a message from a government body
they have never heard from.

THE COUNCIL IS A DIFFERENT BODY from the platform it runs, so it carries no
quotes. Its English name was wrong in this codebase until 2026-08-26 — written
"Emirati Human Development Council", dropping "Resources" — and no test could
have caught that, because nothing here knows an organisation's real name. It
surfaced only because a message delivered to an EXTERNAL address renders the
Exchange signature, so our sign-off and theirs appeared one above the other and
disagreed. The Arabic الموارد is plural, which settles the English.
"""

PLATFORM_NAME_EN = '"Emirati" Human Development Platform'
PLATFORM_NAME_AR = 'منصة "إماراتي" للتنمية البشرية'

COUNCIL_NAME_EN = 'Emirati Human Resources Development Council'
COUNCIL_NAME_AR = 'مجلس تنمية الموارد البشرية الإماراتية'

#: Separates the Arabic half of a bilingual message from the English half.
BILINGUAL_RULE = '───────────────────────────────'
